From 80542e5cc9bf04267e8776b4a77262f8930f0d4a Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Tue, 27 Feb 2018 20:10:02 -0800
Subject: [PATCH] TRITON-193 TRITON-185 broke sapi startup: "Uncaught Error:
 ENOENT: no such file or directory, open
 '/opt/smartdc/sapi/lib/server/../../package.json'" Reviewed by: Todd Whiteman
 <todd.whiteman@joyent.com> Approved by: Todd Whiteman
 <todd.whiteman@joyent.com>

---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index 48c0103..5a37add 100644
--- a/Makefile
+++ b/Makefile
@@ -95,6 +95,7 @@ service: all $(SMF_MANIFESTS)
 	@touch $(SVC_PKGDIR)/site/.do-not-delete-me
 	cp -r $(TOP)/server.js \
 		$(TOP)/node_modules \
+		$(TOP)/package.json \
 		$(SVC_INSTDIR)/
 	cp -r $(TOP)/lib/common \
 		$(TOP)/lib/server \
-- 
2.21.0

